dev-python/pyilmbase/pyilmbase-2.5.2.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+CMAKE_ECLASS=cmake
+inherit cmake-multilib python-r1
+
+DESCRIPTION="IlmBase Python bindings"
+HOMEPAGE="https://www.openexr.com"
+SRC_URI="https://github.com/AcademySoftwareFoundation/openexr/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0/25"
+KEYWORDS="~amd64 ~x86"
+IUSE="exceptions +numpy test"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	>=dev-libs/boost-1.62.0-r1:=[python(+),${MULTILIB_USEDEP},${PYTHON_USEDEP}]
```

Comment:
`python(+)` does not appear to make sense for me. At least all versions since dev-libs/boost-1.62.0-r1 including the current one include that `IUSE=python`, and I believe previous did as well. (I did not check all of them, but at least 1.56 had it as well)